eng/helix/content/InstallNode.ps1

```diff
@@ -0,0 +1,48 @@
+
+param(
+    [Parameter(Mandatory = $true)]
+    $Version
+)
+
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue' # Workaround PowerShell/PowerShell#2138
+
+Set-StrictMode -Version 1
+
+if ((Get-Command "node.exe" -ErrorAction SilentlyContinue)) 
+{ 
+    Write-Host "Found node.exe in PATH"
+    exit
+}
+
+if (Test-Path "%HELIX_CORRELATION_PAYLOAD%\node\bin\node.exe")
+{
+    Write-Host "Node.exe found at %HELIX_CORRELATION_PAYLOAD%\node\bin"
+    exit
+}
+
+$nodeFile="node-v$Version-win-x64"
+$url="http://nodejs.org/dist/v$Version/$nodeFile.zip"
+Write-Host "Starting download of NodeJs ${Version} from $url"
+Invoke-WebRequest -UseBasicParsing -Uri "$url" -OutFile "nodejs.zip"
+Write-Host "Done downloading NodeJS ${Version}"
+
+if ((Get-Command "Expand-Archive" -ErrorAction SilentlyContinue)) 
+{ 
+    Expand-Archive "nodejs.zip"
+}
+else
+{
+    Write-Host "Expand-Archive not found"
+    $shell = new-object -com shell.application
```

Comment:
Try this instead:

```ps1
            if (Get-Command -Name 'Microsoft.PowerShell.Archive\Expand-Archive' -ErrorAction Ignore) {
                # Use built-in commands where possible as they are cross-plat compatible
                Microsoft.PowerShell.Archive\Expand-Archive -Path $tmpfile -DestinationPath $destPath
            }
            else {
                # Fallback to old approach for old installations of PowerShell
                Add-Type -AssemblyName System.IO.Compression.FileSystem
                [System.IO.Compression.ZipFile]::ExtractToDirectory($tmpfile, $korebuildPath)
            }
```

source: https://github.com/aspnet/BuildTools/blob/ec244eafdc277b29813c81e19e9e9768cad8839a/scripts/bootstrapper/run.ps1#L116-L124